src/box/sql/where.c

```diff
@@ -1233,9 +1233,9 @@ whereRangeSkipScanEst(Parse * pParse,		/* Parsing & code generating context */
 	int rc = 0;
 	enum field_type type = p->key_def->parts[nEq].type;
 
-	sql_value *p1 = 0;	/* Value extracted from pLower */
-	sql_value *p2 = 0;	/* Value extracted from pUpper */
-	sql_value *pVal = 0;	/* Value extracted from record */
+	struct Mem *p1 = 0;	/* Value extracted from pLower */
```

Comment:
Please use
```
/* Value extracted from pLower. */
struct Mem *p1 = NULL;
```
 instead of
```sql_value *p1 = 0;	/* Value extracted from pLower */```

The same for p2 and pVal.